x-pack/test/lists_api_integration/security_and_spaces/tests/delete_exception_list_items.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { ExceptionListItemSchema } from '../../../../plugins/lists/common';
+import { getExceptionListItemResponseMockWithoutAutoGeneratedValues } from '../../../../plugins/lists/common/schemas/response/exception_list_item_schema.mock';
+import {
+  getCreateExceptionListItemMinimalSchemaMock,
+  getCreateExceptionListItemMinimalSchemaMockWithoutId,
+} from '../../../../plugins/lists/common/schemas/request/create_exception_list_item_schema.mock';
+import { getCreateExceptionListMinimalSchemaMock } from '../../../../plugins/lists/common/schemas/request/create_exception_list_schema.mock';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import {
+  EXCEPTION_LIST_URL,
+  EXCEPTION_LIST_ITEM_URL,
+} from '../../../../plugins/lists/common/constants';
+
+import { deleteAllExceptions, removeExceptionListItemServerGeneratedProperties } from '../../utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  describe('delete_exception_list_items', () => {
+    describe('delete exception list items', () => {
+      afterEach(async () => {
+        await deleteAllExceptions(es);
+      });
+
+      it('should delete a single exception list item by its item_id', async () => {
+        // create an exception list
+        await supertest
+          .post(EXCEPTION_LIST_URL)
+          .set('kbn-xsrf', 'true')
+          .send(getCreateExceptionListMinimalSchemaMock())
+          .expect(200);
+
+        // create an exception list item
+        await supertest
+          .post(EXCEPTION_LIST_ITEM_URL)
+          .set('kbn-xsrf', 'true')
+          .send(getCreateExceptionListItemMinimalSchemaMock())
+          .expect(200);
+
+        // delete the exception list item by its item_id
+        const { body } = await supertest
+          .delete(
+            `${EXCEPTION_LIST_ITEM_URL}?item_id=${
+              getCreateExceptionListItemMinimalSchemaMock().item_id
+            }`
+          )
+          .set('kbn-xsrf', 'true')
+          .expect(200);
+
+        const bodyToCompare = removeExceptionListItemServerGeneratedProperties(body);
+        expect(bodyToCompare).to.eql(getExceptionListItemResponseMockWithoutAutoGeneratedValues());
+      });
+
+      it('should delete a single exception list item using an auto generated id', async () => {
+        // create an exception list
+        await supertest
+          .post(EXCEPTION_LIST_URL)
+          .set('kbn-xsrf', 'true')
+          .send(getCreateExceptionListMinimalSchemaMock())
+          .expect(200);
```

Comment:
These simple blocks get repeated a bit -- 19 for this one. From a readability perspective it may be worth breaking this out as a helper, thoughts? It's nice having the whole block when needing to modify tests, but these seem like pretty simple operations to encapsulate for readability.

Reply:
Yeah, was debating that. Turning that into a one-liner.

It's boilerplatey and we might with a follow up with doing that here. I won't do it with this one as we have other PR's that are about to go through where we will need these tests in place for some protection against obvious oopsies but down the road I think we should here and probably in detection engine where things repeat.

Appreciate the thoughts.